RecoJets/JetProducers/python/hltMVAJetPuId_cff.py

```diff
@@ -0,0 +1,15 @@
+from RecoJets.JetProducers.hltPUIdAlgo_cff import *
+
+hltMVAJetPuIdCalculator = cms.EDProducer('MVAJetPuId',
```

Comment:
there is no MVAJetPuId EDProducer. There is a MVAJetPuIdProducer.
Do you have a working configuration that actually runs?